.github/workflows/test.yml

```diff
@@ -53,14 +53,9 @@ jobs:
         restore-keys: |
           ${{ runner.os }}-pip-
 
-    - name: Install python dependencies
-      run: |
-        pip install -r ./requirements.txt
-        pip install -r ./tests/tests-requirements.txt
-
```

Comment:
tests requirements need to be reintroduced otherwise tests will fail due to missing deps